snapshot/lib/snapshot/runner.rb

```diff
@@ -204,6 +207,21 @@ def erase_simulator(device_type)
       `xcrun simctl erase #{device_udid} &> /dev/null`
     end
 
+    def localize_simulator(device_type, language, locale)
+      apple_locale = language.sub("-", "_")
+      apple_languages = [language]
+
+      UI.message "Localizing #{device_type} (AppleLocale=#{apple_locale} AppleLanguages=[#{apple_languages}])"
+
+      device_udid = TestCommandGenerator.device_udid(device_type)
+
+      plist = Hash[]
+      plist[:AppleLocale] = apple_locale
+      plist[:AppleLanguages] = apple_languages
```

Comment:
Thanks, will do. (It has been many years since I worked with Ruby so all hints appreciated.)